chris_backend/pipelines/models.py

```diff
@@ -80,7 +81,7 @@ def get_accesible_pipelines(user):
         """
         queryset = Pipeline.objects.all()
         # if the user is chris then return all the pipelines in the queryset
-        if user.username == 'chris':
+        if user.username == 'chris' or not user.is_authenticated:
```

Comment:
Here, we have a special case where the "chris" user is able to see all pipelines.

This is not what we want if the user is not authenticated. Instead, we should change the lookup on line 87 such that for anonymous user, return all pipelines where `locked=False`.

Reply:
Explanation: `locked` is a poorly named attribute, think of it as meaning whether or not a pipeline is private.